### PR TITLE
Queue worker should update stock level with checker

### PR DESCRIPTION
**Pull request by ransomweaver**

### CODE REVIEW

**Comment by steveoliver** [comment]
It looks like this fail may be related to the failing test introduced in https://travis-ci.org/BBGuy/commerce_stock/jobs/194670752.  I'll see if I can get that issue fixed, and also review this, whether or not it is related.

**Comment by steveoliver** [comment]
This will pass once the test failure follow-ups in https://github.com/BBGuy/commerce_stock/pull/16 and #10 are done.

**Comment by steveoliver** [comment]
See https://www.drupal.org/node/2849275

**Comment by BBGuy** [comment]
#16 and #10 are committed. Can review this tomorrow. well done

**Comment by BBGuy** [comment]
ransomweaver this is not committed only the two blocking PRs

**Comment by steveoliver** [comment]
I actually think the updateLocationStockLevel should move from the local checker to the local updater.  See https://www.drupal.org/node/2842583 for the discussion on a local updater interface.

**Comment by steveoliver** [comment]
Let's repurpose this issue to be a follow-up to the split issue, and just move the update method to the updater ... it was missed in the split.  Then this current patch will be unnecessary.

File: modules/local_storage/src/Plugin/QueueWorker/StockLevelUpdater.php [hunk 1]
```diff
@@ -22,10 +22,10 @@ class StockLevelUpdater extends QueueWorkerBase {
    */
   public function processItem($data) {
     $entity_id = $data;
-    $updater = \Drupal::service('commerce_stock.local_stock_service')->getStockUpdater();
-    $locations = $updater->getLocationList(TRUE);
+    $checker = \Drupal::service('commerce_stock.local_stock_service')->getStockChecker();
```
Comment (BBGuy) [inline modules/local_storage/src/Plugin/QueueWorker/StockLevelUpdater.php:25]: getLocationList() is on the checker and updateLocationStockLevel() needs to move to the updater. As a result this function should use both.
I think we may handle this in https://github.com/BBGuy/commerce_stock/pull/19.

**Comment by steveoliver** [comment]
This is handled in #19.  We can close this now.

**Pull request closed without merging**